packages/itwin/tree-widget/src/components/TreeWidgetUiItemsProvider.tsx

```diff
@@ -2,30 +2,19 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-/* eslint-disable react/display-name */
 
 import React from "react";
 import { StagePanelLocation, StagePanelSection, StageUsage, UiItemsProvider, Widget } from "@itwin/appui-react";
-import { SelectableContentDefinition } from "@itwin/components-react";
 import { TreeWidget } from "../TreeWidget";
-import { CategoriesTreeComponent, CategoriesTreeComponentProps } from "./trees/category-tree/CategoriesTreeComponent";
-import { ClassGroupingOption } from "./trees/Common";
-import { ModelsTreeComponent, ModelTreeComponentProps } from "./trees/models-tree/ModelsTreeComponent";
-import { TreeWidgetComponent } from "./TreeWidgetComponent";
+import { CategoriesTreeComponent } from "./trees/category-tree/CategoriesTreeComponent";
+import { ModelsTreeComponent } from "./trees/models-tree/ModelsTreeComponent";
+import { TreeDefinition, TreeWidgetComponent } from "./TreeWidgetComponent";
 
 export interface TreeWidgetOptions {
   defaultPanelLocation?: StagePanelLocation;
   defaultPanelSection?: StagePanelSection;
   defaultTreeWidgetPriority?: number;
-  enableElementsClassGrouping?: boolean;
-  additionalTrees?: SelectableContentDefinition[];
-  modelsTreeProps?: ModelTreeComponentProps;
-  categoriesTreeProps?: CategoriesTreeComponentProps;
-  defaultTreeId?: string;
```

Comment:
is the recommended approach moving fwd going to be, if you want to set a specific tree as the default, or add an additional tree, or change any of the defaults for our default trees here, you need to specify all that yourself?

Reply:
Yes. I think this way provides the most flexibility for consumers.